nupic/algorithms/anomaly.py

```diff
@@ -123,25 +126,9 @@ def compute(self, activeColumns, predictedColumns,
                               	(used in anomaly-likelihood)
     @return the computed anomaly score; float 0..1
```

Comment:
Vitaly explained this to me.

@breznak - Since you integrated this class into the CLAModel, the output must still use the same likelihood logic. We can't change the CLAModel behavior without a thorough evaluation of the anomaly detection performance.